### DISCUSSION TITLE
Bagaiman Cara untuk mengubah hasil predict yang masih ter-Scale

### DISCUSSION THREAD

**Discussion by rulierinton** [teamalgoritma/community#156] [Q&A] [answered]
Halo Bapak/Ibu,

Saya sedang mengerjakan Capstone akan tetapi saya mengalami kendala kareana data hasil dari function Predict masih ter-scale sperti gambar yang saya cantumkan, column yang dimaksud adalah strength.
Sebelum melakukan predict saya membuat model dengan menggunakan data train yang saya scale terlebih dahulu bagaimana cara agar hasil predict tidak dalam bentuk scale.

Terimakasih

![Scale](https://user-images.githubusercontent.com/101322908/175501840-1fbabd5f-5f5e-4f16-bb2c-fb885394744c.PNG)
,

**Comment by kevbow** [comment, marked as answer]
Halo @rulierinton,

Apakah target variabel sebelumnya dilakukan scaling juga? Jika iya maka tahapan scaling di awal perlu diubah. Target variabel tidak perlu dilakukan scaling. 
Demikian yang dapat kami sampaikan, terima kasih.

**Reply by rulierinton** [reply]
iyaa betul pak @kevbow saya scaling juga untuk variable targetnya, baik akan saya coba lakukan yang bapak sarankan.

terimaksih atas bantuannya pak
